### PR TITLE
[IMP] account: bank account configuration, remove views

### PR DESCRIPTION
**Pull request by william-andre**
Task 1966321

There are 3 different ways to configure a bank account and it’s confusing:
- config bar wizard
- journal
- bank account (Menu: Configuration/Accounting/Bank accounts)

We want to use the wizard for creation and the journal for configuration, so removing the bank account (Menu: Configuration/Accounting/Bank accounts) view).

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/point_of_sale/views/account_statement_view.xml [hunk 1]
```diff
@@ -19,16 +19,4 @@
...
             </field>
         </record>
-        <record id="view_account_bank_journal_form_inherited_pos" model="ir.ui.view">
-            <field name="name">account.bank.journal.form.inherit.pos</field>
-            <field name="model">account.journal</field>
-            <field name="inherit_id" ref="account.view_account_bank_journal_form"/>
-            <field name="arch" type="xml">
-                <xpath expr="//div[@name='advanced_info']" position="inside">
-                    <group string="Point of Sale">
-                        <field name="journal_user"/>
```
Comment (csnauwaert) [inline addons/point_of_sale/views/account_statement_view.xml]: same question here, we loose the journal_user field, is it okay? is it intended? Should we not set it on another view?

File: addons/pos_mercury/views/pos_mercury_views.xml [hunk 2]
```diff
@@ -80,16 +80,6 @@
                 </xpath>
             </field>
         </record>
-        <record id="view_account_bank_journal_form_inherited_pos_mercury" model="ir.ui.view">
-            <field name="name">account.bank.journal.form.inherited.pos.mercury</field>
-            <field name="model">account.journal</field>
-            <field name="inherit_id" ref="point_of_sale.view_account_bank_journal_form_inherited_pos"></field>
-            <field name="arch" type="xml">
-                <xpath expr="//field[@name='journal_user']" position="after">
-                    <field name="pos_mercury_config_id" attrs="{'invisible': [('journal_user', '=', False)]}"/>
```
[the remark this answers is not in the document]
Comment (william-andre) [reply addons/pos_mercury/views/pos_mercury_views.xml]: it is still in `pos_mercury.view_account_journal_pos_user_form`

File: addons/point_of_sale/views/account_statement_view.xml (hunk 1 above)
Comment (william-andre) [reply addons/point_of_sale/views/account_statement_view.xml]: it is still in `point_of_sale.view_account_journal_pos_user_form`

File: addons/pos_mercury/views/pos_mercury_views.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (csnauwaert) [reply addons/pos_mercury/views/pos_mercury_views.xml]: good then :)

File: addons/point_of_sale/views/account_statement_view.xml (hunk 1 above)
Comment (csnauwaert) [reply addons/point_of_sale/views/account_statement_view.xml]: good then :+1:

File: addons/l10n_ch/views/res_bank_view.xml [hunk 3]
```diff
@@ -23,16 +23,5 @@
...
             </field>
         </record>
-
-        <record id="isr_bank_journal_form" model="ir.ui.view">
-            <field name="name">l10n_ch.bank.journal.form</field>
-            <field name="model">account.journal</field>
-            <field name="inherit_id" ref="account.view_account_bank_journal_form"/>
-            <field name="arch" type="xml">
-                <xpath expr="//field[@name='bank_acc_number']" position="after">
-                    <field name="l10n_ch_postal"/>
```
[the remark this answers is not in the document]
Comment (william-andre) [reply addons/l10n_ch/views/res_bank_view.xml]: I moved it to the default view for `account.journal`

**Comment by qdp-odoo** [comment]
@robodoo r+

**Pull request closed without merging**